Ghidra/Features/Python/Module.manifest

```diff
@@ -1,4 +1,4 @@
-DATA SEARCH IGNORE DIR: jython-2.7.0
+DATA SEARCH IGNORE DIR: jython-2.7.2
```

Comment:
noticed this is jython-2.7.0 and not jython-2.7.1   was that supposed to be jython-2.7.1 this whole time

Reply:
Yes, but it only affects efficiency.